CoreMapView/src/au/gov/asd/tac/constellation/views/mapview2/plugins/ExtractCoordsFromGraphPlugin.java

```diff
@@ -0,0 +1,235 @@
+                    // Ids for all attributes needed from a single vertext of a graph
+                    int lonID = GraphConstants.NOT_FOUND;
+                    int latID = GraphConstants.NOT_FOUND;
+                    int colourID = GraphConstants.NOT_FOUND;
+                    int blazeID = GraphConstants.NOT_FOUND;
+                    int overlayID = GraphConstants.NOT_FOUND;
+                    int labelAttrID = GraphConstants.NOT_FOUND;
+                    int identifierID = GraphConstants.NOT_FOUND;
+
+                    int elementCount;
+
+                    switch (elementType) {
+                        case VERTEX:
+                            // Get IDs
+                            lonID = SpatialConcept.VertexAttribute.LONGITUDE.get(graph);
+                            latID = SpatialConcept.VertexAttribute.LATITUDE.get(graph);
+                            colourID = VisualConcept.VertexAttribute.COLOR.get(graph);
+                            blazeID = VisualConcept.VertexAttribute.BLAZE.get(graph);
+                            overlayID = VisualConcept.VertexAttribute.OVERLAY_COLOR.get(graph);
+                            labelAttrID = VisualConcept.VertexAttribute.LABEL.get(graph);
+                            identifierID = VisualConcept.VertexAttribute.IDENTIFIER.get(graph);
+                            elementCount = graph.getVertexCount();
+                            break;
+                        case TRANSACTION:
+                            lonID = SpatialConcept.VertexAttribute.LONGITUDE.get(graph);
+                            latID = SpatialConcept.VertexAttribute.LATITUDE.get(graph);
+                            elementCount = graph.getTransactionCount();
+                            break;
+                        default:
+                            continue;
+                    }
+
+                    // Loop though every graph element
+                    for (int elementPos = 0; elementPos < elementCount; ++elementPos) {
+                        int elementID = -99;
+
+                        switch (elementType) {
+                            case VERTEX:
+                                elementID = graph.getVertex(elementPos);
+
+                                break;
+                            case TRANSACTION:
+                                elementID = graph.getTransaction(elementPos);
+                                break;
+                            default:
+                                break;
+                        }
+
+                        // For all the vertices
+                        if (lonID != GraphConstants.NOT_FOUND && latID != GraphConstants.NOT_FOUND && elementType == GraphElementType.VERTEX && elementID != -99) {
+                            // Get lattidue and longitude
+                            final float elementLat = graph.getObjectValue(latID, elementID);
+                            final float elementLon = graph.getObjectValue(lonID, elementID);
```

Comment:
since these 2 attributes are both of type Float Object, you should probably make these types `Float` (in case the values are null so they're not confused with those at point 0.0)